setup.py

```diff
@@ -33,37 +33,16 @@
     'cirq.' + package for package in find_packages(where='cirq')
 ]
 
-setup(
-    name='cirq',
-    version=__version__,
-    url='http://github.com/quantumlib/cirq',
-    author='The Cirq Developers',
-    author_email='cirq@googlegroups.com',
-
-    # CAUTION: the semantics of python_requires and how it interacts with PYPI
-    # are extremely inconvenient, so read this before changing it.
-    #
-    # One would assume that, because each wheel within a package specifies a
-    # python_requires line, PYPI would consider the python_requires of the
-    # package to be the union of each of its wheels. This is not the case. What
-    # PYPI actually does is assert that the python_requires of the package is
-    # the python_requires of the *first wheel uploaded to the package*. So if
-    # you have a wheel targeting python 2, and set "python_requires='2.7.*'"
-    # for that wheel, then it doesn't matter how many python 3 wheels you add to
-    # the package; python 3 users will not be able to pip install them.
-    #
-    # The workaround for this problem is to set the actual python_requires of
-    # all wheels in a package to the union of the desired python_requires of all
-    # the wheels in the package (or to not set it at all). Then, when uploading
-    # wheels, ensure that their names encode the version they are targeting. For
-    # example, a wheel named 'cirq-#.#.#-py27-none-any.whl' will only be
-    # installed in python 2.7.
-    python_requires=('>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*, !=3.5.0, '
-                     '!=3.5.1'),
-
-    install_requires=requirements,
-    license='Apache 2',
-    description=description,
-    long_description=long_description,
-    packages=cirq_packages,
-    package_data={'cirq.api.google.v1': ['*.proto']})
+setup(name='cirq',
+      version=__version__,
+      url='http://github.com/quantumlib/cirq',
+      author='The Cirq Developers',
+      author_email='cirq@googlegroups.com',
+      python_requires=('!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*, !=3.5.0, '
```

Comment:
This technically allows python 1...

Change it to just be `>=3.5.2`.

Reply:
PTAL